midp/gfx.js

```diff
@@ -188,12 +188,10 @@ var currentlyFocusedTextEditor;
                 setImageData(imageData, img.naturalWidth, img.naturalHeight, context);
 
                 URL.revokeObjectURL(img.src);
-                img.src = '';
                 resolve();
             }
             img.onerror = function(e) {
                URL.revokeObjectURL(img.src);
-               img.src = '';
```

Comment:
It's weird that Chrome would choke on these, but they shouldn't be necessary, as @yxl noted in another PR recently, so it's fine to remove them.